build.zig

```diff
@@ -33,7 +33,7 @@ pub fn build(b: *std.Build) anyerror!void {
         "-serial",             "mon:stdio",
     });
 
-    qemu_cmd.step.dependOn(&kernel.step);
+    qemu_cmd.step.dependOn(b.getInstallStep());
```

Comment:
I'm not super familiar with the Zig build system, this is how I've done it with other projects in the past and it has worked reliably. Not sure why `kernel.step` did not work originally.

Reply:
`kernel.step` refers to the build step, the output of which is copied to the install dir (i.e. `zig-out/bin/`) which is where `qemu_cmd` expects the binary to be.

Reply:
Ah okay cool thanks for explaining.

Reply:
I guess if `kernel.step` was working previously, it was by accident. Depending on the install step is the correct way.